common/src/main/resources/data/sql/migration/V1-V2/mysql.sql

```diff
@@ -0,0 +1,9 @@
+-- MySQL Conversion from V1 -> V2
+
+-- Changes Needed:
+-- 1. Create Version Table
+
+-- Journey Database Version Tracker
+
+CREATE TABLE journey_db_version (db_version int, INDEX journey_db_version_idx (db_version));
+INSERT INTO journey_db_version (db_version) VALUES (2);
```

Comment:
Nitpick, but can you make this a similar format to the schema files? Newlines for each parameter, capitalized types, etc